css/style.css

```diff
@@ -117,3 +118,39 @@ a {
     margin-right: 15px;
     padding-right: 15px;
   }
+
+  
+  @media(min-width:400px) {
```

Comment:
make sure to put @media and its properties at the bottom.

CSS goes from top to bottom. by having @media here makes this the default size and properties for the site.